Sources/Common/Core/CellArray/index.d.ts

```diff
@@ -1,5 +1,5 @@
-/// <reference path='../../../types.ts' />
-import vtkDataArray from "vtk.js/Sources/Common/Core/DataArray";
+import { TypedArray } from "../../../types";
+import vtkDataArray from "../../Core/DataArray";
```

Comment:
I think all of your relative paths in this commit need to remove a single ".." traversal. So, this should be:

```
import { TypedArray } from "../../types";
import vtkDataArray from "../Core/DataArray";
```

Prior to the change: 

![image](https://user-images.githubusercontent.com/1812167/119492103-35a47080-bd4e-11eb-9386-3022ea30bad5.png)

After removing a ".." traversal:

![image](https://user-images.githubusercontent.com/1812167/119492165-46ed7d00-bd4e-11eb-9b21-f30740c10250.png)

Reply:
The UMD build does have folders structures any idea why it's not the case for the ESM build ?

![image](https://user-images.githubusercontent.com/878518/119550762-13841000-bd99-11eb-81d4-d81a1897db4e.png)

Reply:
Ah right, that will explain the inconsistency.

The ESM build pulled the JS files out of the folder due to import issues with consuming projects that use Rollup. Specifically, rollup wouldn't resolve stuff like `import vtkDataArray from '@kitware/vtk.js/Common/Core/DataArray'` into using `Common/Core/DataArray/index.js`. I'd have to double check to see if this is now doable in rollup, since having consistent paths for the d.ts files would be nice. The fallback would be to rewrite paths in the d.ts files when generating the ESM build.

Reply:
For now, I will make changes to the ESM package build script so it can rewrite the d.ts files to reference the appropriate path. @daker, if you want to take a shot at that, you can check out the npm script `release:create-packages`. Otherwise, I can merge this now and make the build script changes later.

Reply:
Yes just merge for now so i can continue adding more, i will also check the npm script